api_routes.py

```
@@ -6,8 +6,10 @@
 from datetime import datetime
 from typing import Dict, Any, List
 import logging
+import math
```

Ruff (0.12.2)

`math` imported but unused

Remove unused import: `math`

(F401)